src/combinatorics/simple_graph/basic.lean

```diff
@@ -480,6 +504,29 @@ set.to_finset_card _
   (univ : finset G.edge_set).card = G.edge_finset.card :=
 fintype.card_of_subtype G.edge_finset (mem_edge_finset _)
 
+section from_edge_finset
+
+variable (s : finset (sym2 V))
+
+/--
+`from_edge_finset` constructs a `simple_graph` from a finset of edges, without loops.
+-/
+def from_edge_finset : simple_graph V := from_edge_set s
+
+@[simp] lemma from_edge_finset_adj : (from_edge_finset s).adj v w ↔ v ≠ w ∧ ⟦(v, w)⟧ ∈ s := iff.rfl
+
+@[simp] lemma from_edge_finset_edge_finset [fintype G.edge_set] :
+  from_edge_finset (G.edge_finset) = G :=
+by simp only [from_edge_finset, coe_edge_finset, from_edge_set_edge_set]
+
+@[simp] lemma from_edge_finset_empty : from_edge_finset (∅ : finset (sym2 V)) = ⊥ :=
+  by simp only [from_edge_finset, coe_empty, from_edge_set_empty]
+
+@[simp] lemma from_edge_finset_univ [fintype V] : from_edge_finset (univ : finset (sym2 V)) = ⊤ :=
+  by simp only [from_edge_finset, coe_univ, from_edge_set_univ]
+
```

Comment:
These are also unnecessary because they follow directly from simp lemmas:
```lean
@[simp] lemma from_edge_finset_empty :
  from_edge_set ((∅ : finset (sym2 V)) : set (sym2 V)) = ⊥ :=
by simp

@[simp] lemma from_edge_finset_univ [fintype V] :
  from_edge_set ((univ : finset (sym2 V)) : set (sym2 V)) = ⊤ :=
by simp
```

Reply:
Not sure what you mean, Kyle, because your lemma names don't match the statements.

Reply:
@YaelDillies You should interpret this in the context that I'm suggesting to remove `from_edge_finset`; the lemmas specialized to finsets are not necessary if it is removed (this also serves as evidence that this finset definition is not necessary).

Reply:
Removed the `from_edge_finset` section, so this is no longer relevant.